files/zh-tw/glossary/block-level_content/index.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: 區塊級元素
-slug: Web/HTML/Block-level_elements
+slug: Glossary/Block-level_content
+original_slug: Web/HTML/Block-level_elements
 ---
 
 {{HTMLSidebar}}
```

Comment:
remove {{HTMLSidebar}} as of https://github.com/mdn/content/pull/25891/files#diff-1b8c7b8410ecdfac9be52b31d2b6a9da4476270803586cab6963251e1b2e8c06

```suggestion
```